CHANGELOG.md

```diff
@@ -2,533 +2,534 @@
 
 ## master / unreleased
 
-* [FEATURE] Ruler: Add `external_labels` option to tag all alerts with a given set of labels.
```

Comment:
seems that those changes were not intentional

Reply:
Oh this one is intentional because FEATURE should go after CHANGE (I as added the PR number)

Reply:
I will fix the unintended change (from '*' to '-' four bullets)